code/modules/projectiles/gun.dm

```diff
@@ -107,7 +107,7 @@
 				slot_s_store_str = icon,
 			)
 		item_icons = item_icons_cache[type]
-	if(one_hand_penalty && (!wielded_item_state))//If the gun has a one handed penalty but no wielded item state then use this generic one.
+	if(one_hand_penalty || twohanded && (!wielded_item_state))//If the gun has a one hand penalty or is twohanded but no wielded item state then use this generic one.
```

Comment:
Why did you change my comment to be grammatically incorrect?

Reply:
I fucked up and accidentally deleted the "one handed" part to write the twohanded thing. Sorry. Does it really matter that I fix it? I kinda don't want to.

Reply:
Yeah it matters, you made the comment less readable by butchering the grammar that was already correct.

Reply:
Fine, fuck me

Reply:
Fixed the typo, and also fixed a typo YOU made in the original comment, you fucking weenie.